pkg/build/info.go

```diff
@@ -94,6 +96,24 @@ func (i *Info) expandManifestBuildArgs(previousImageArgs map[string]string) (err
 	return nil
 }
 
+func (i *Info) expandSecrets() (err error) {
+	for k, v := range i.Secrets {
+		val := v
+		if strings.HasPrefix(v, "~/") {
```

Comment:
should we check also that `~/` is at the begginnig?

Reply:
That should be ensured by using `strings.HasPrefix`, no? What am I missing? 🤔